File: src/com/ptby/dynamicreturntypeplugin/config/ClassMethodConfigList.java
package com.ptby.dynamicreturntypeplugin.config;

import java.util.ArrayList;

public class ClassMethodConfigList extends ArrayList<ClassMethodConfigKt>{
    public ClassMethodConfigList( ClassMethodConfigKt... classMethodConfigs ){
        for ( ClassMethodConfigKt classMethodConfig : classMethodConfigs ) {
            add( classMethodConfig );
        }
    }
}

File: src/main/kotlin/com/ptby/dynamicreturntypeplugin/config/ClassMethodConfigList.kt
package com.ptby.dynamicreturntypeplugin.config

import java.util.*

class ClassMethodConfigList(vararg classMethodConfigs: ClassMethodConfigKt) : ArrayList<ClassMethodConfigKt>() {
    init {
        for (classMethodConfig in classMethodConfigs) {
            add(classMethodConfig)
        }
    }
}
